src/ZODB/tests/StorageTestBase.py

```diff
@@ -113,18 +113,24 @@ def handle_all_serials(oid, *args):
 
     The original interface just returned the serialno for the
     object.
+
+    The updated multi-commit API returns nothing from store(), and
+    returns a sequence of resolved oids from tpc_vote.
     """
     d = {}
     for arg in args:
         if isinstance(arg, bytes):
             d[oid] = arg
-        elif arg is None:
-            pass
-        else:
-            for oid, serial in arg:
-                if not isinstance(serial, bytes):
-                    raise serial # error from ZEO server
-                d[oid] = serial
+        elif arg:
+            for t in arg:
+                if isinstance(t, bytes):
+                    # This will be the tid returned by tpc_finish.
```

Comment:
It could also be an oid that's returned by vote to indicate that a conflict was resolved. This is important in some situations, but perhaps not in cases that are tested here.

This is a mess (like a lot of ZODB test infrastructure).  For this function to be "correct", it needs to consider where the data came from.